source/_components/cover.rpi_gpio.markdown

```diff
@@ -27,6 +27,8 @@ To enable Raspberry Pi Covers in your installation, add the following to your `c
 # Example configuration.yaml entry
 cover:
   platform: rpi_gpio
+  relay_time: 0.2
+  state_pull_mode: 'UP'
```

Comment:
They're already added to the Configuration variables, but it wasn't clear where 'relay_time' and 'state_pull_mode' were supposed to go in the configuration. Essentially leading us on a multi hour long troubleshooting. 

We had been trying to add them like this:

```
cover:
  platform: rpi_gpio
  covers:
    - relay_pin: 10
      state_pin: 11
    - relay_pin: 12
      state_pin: 13
      name: 'Right door'
```

But the configuration actually calls for them to to be like this:
```
cover:
  platform: rpi_gpio
  relay_time: 0.2
  state_pull_mode: 'UP'
  covers:
    - relay_pin: 10
      state_pin: 11
    - relay_pin: 12
      state_pin: 13
      name: 'Right door'
```

How do you suggest we avoid that confusion for others?